pkg/core/proxy/integrations/mongo/operation.go

```diff
@@ -475,6 +477,128 @@ func extractSectionSingle(data string) (string, error) {
 	return content, nil
 }
 
+func processOpReply(expected, mongoRequest models.MongoRequest, replySpec *models.MongoOpReply, logger *zap.Logger) string {
+	if len(replySpec.Documents) == 0 {
+		return ""
+	}
+	for _, responseSection := range replySpec.Documents {
```

Comment:
does opReply has sections..?

Reply:
no it just have documents 
```
type opReply struct {
	reqID        int32
	flags        wiremessage.ReplyFlag
	cursorID     int64
	startingFrom int32
	numReturned  int32
	documents    []bsoncore.Document
}
```

Reply:
may be them change the name of responseSection